lib/schema.js

```diff
@@ -19,6 +19,43 @@ const {
 
 const factorify = Symbol('factorify');
 const validate = Symbol('validate');
+const classifyDomains = Symbol('classifyDomains');
```

Comment:
This should be moved to `schema-processors` and renamed to `processDomains` to be consistent.